src/index.html

```diff
@@ -1,12 +1,228 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="page__header header">
+
+      <a class="header__logo logo" href="/"><img src="./images/logo.png" alt="logo"></a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a class="nav__link is-active" href="#">Apple</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">Samsung</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">Смартфоны и телефоны</a></li>
+          <li class="nav__item"><a class="nav__link" href="#" data-qa="nav-hover">Ноутбуки и компьютеры</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">Гаджеты</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">Планшеты</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">Фото</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">Видео</a></li>
+        </ul>
+      </nav>
+
+    </header>
+
+    <main>
+      <div class="page__cards cards">
+
+        <div class="cards__product product" data-qa="card">
+          <img class="product__card-image card-image" src="./images/imac.jpeg" alt="APPLE A1419 iMac">
+          <h2 class="product__card-title card-title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+          <p class="product__card-code card-code">Код товара: 195434</p>
+
+          <div class="product__card-rating card-rating">
+            <div class="card-rating__stars">
+              <div class="card-rating__star"></div>
+              <div class="card-rating__star"></div>
+              <div class="card-rating__star"></div>
+              <div class="card-rating__star"></div>
+              <div class="card-rating__star"></div>
+            </div>
+            <p class="card-rating__reviews">Отзывов: 5</p>
+          </div>
+
+          <div class="product__price-container price-container">
+            <p class="price-container__text">Цена:</p>
+            <p class="price-container__sum">69 999 грн</p>
+          </div>
+          
+          <a class="product__buy" href="#" data-qa="card-hover">Купить</a>
+        </div>
+        <div class="cards__product product">
+          <img class="product__card-image card-image" src="./images/imac.jpeg" alt="APPLE A1419 iMac">
```

Comment:
if more than 2-3 attributes move them to the new line

```suggestion
          <img
            class="product__card-image card-image"
            src="./images/imac.jpeg"
            alt="APPLE A1419 iMac"
          >
```